benchmark/long_statment.py

```diff
@@ -0,0 +1 @@
 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z)) * (a*z+3+2*x + 3*y - x/(z**2-4) - x**(y**z))
```

Comment:
I think we should not check in this long file into git.

Reply:
ok, i'll make dir for generating long codes and running it as subprocess.